crates/ironrdp-cliprdr-native/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+/// Native CLIPRDR backend implementations. Currently only Windows is supported.
```

Comment:
Just FYI: For some lints newer `1.72.0` toolchain version is required, I updated it from `1.70.0`. I think It would not hurt.

Also I set `resolver = "2"` in root `Cargo.toml`, because after the update cargo throws this warning:
```
some crates are on edition 2021 which defaults to `resolver = "2"`, but virtual workspaces default to `resolver = "1"
```

Reply:
Sounds good to me!